packages/clerk-js/tsconfig.declarations.json

```diff
@@ -8,6 +8,7 @@
     "declarationMap": false,
     "sourceMap": false,
     "outDir": "./dist",
-    "declarationDir": "./dist/types"
+    "declarationDir": "./dist/types",
+    "noImplicitReturns": false
```

Comment:
We only warn against this in our code (and have some instances where we need to fix this). During compilation then it fails unless turned off. Since we don't enforce it in our normal code I opted for this for now.